packages/odyssey/src/scss/base/_typography-header.scss

```diff
@@ -18,39 +18,44 @@ h3,
 h4,
```

Comment:
Do you think it's worth reviewing the baseline style separations (e.g. `_typography-header.scss`) we inherited from Core's WC work?